frontend/projects/upgrade/src/app/features/dashboard/feature-flags/modals/upsert-feature-flag-modal/upsert-feature-flag-modal.component.ts

```diff
@@ -206,8 +206,8 @@ export class UpsertFeatureFlagModalComponent {
       tags,
       status,
       filterMode,
-      featureFlagSegmentInclusion,
-      featureFlagSegmentExclusion,
+      featureFlagSegmentInclusion: [],
```

Comment:
Instead of always sending an empty array we should not send these properties. This change should also be made to `createAddRequest`.

Reply:
code updated